packages/postcss-merge-rules/src/lib/ensureCompatibility.js

```diff
@@ -13,7 +12,23 @@ const cssInOutOfRange = 'css-in-out-of-range';
 const formValidation = 'form-validation';
 
 /** @type {string[]} */
-const prefixes = vendors.map((v) => `-${v}-`);
+const prefixes = [
+  '-ah-',
+  '-apple-',
+  '-atsc-',
+  '-epub-',
+  '-hp-',
+  '-khtml-',
+  '-moz-',
+  '-ms-',
+  '-o-',
+  '-rim-',
+  '-ro-',
+  '-tc-',
+  '-wap-',
+  '-webkit-',
+  '-xv-',
+];
```

Comment:
Maybe just regexp?

Reply:
I've replaced it with a regex. Unfortunately, I think both with and without regex the code is not particularly clear